pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+
+    //storage-client
+    rpc PutChunks(Chunks) returns (Nothing);
+    //In GetChunks rpc request send buf as nil
+    rpc GetChunks(Chunks) returns (stream Chunks);
+    rpc DeleteChunks(ChunkID) returns (Nothing);
+
+    //table-client
+    rpc ListTables(Nothing) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (Nothing);
+    rpc DeleteTable(TableName) returns (Nothing);
+    rpc DescribeTable(TableName) returns (DescribeTableResponse);
+    rpc UpdateTable(UpdateTableRequest) returns (Nothing);
+
+    //used by storage-client & index-client
+    rpc Stop(Nothing) returns (Nothing);
+}
+
+message Chunks {
+    repeated Chunk chunks = 1;
+}
+
+message Chunk {
+    bytes buf         = 1;
+    string key        = 2;
+    string tableName  = 3;
+}
+
+message ChunksResponse {
+    repeated ChunkResponse chunks = 1;
+}
+
+message ChunkResponse {
+    bytes buf       = 2;
+}
+
+message Nothing {}
+
+message ChunkID {
```

Comment:
Why do we need a message for this? Looks overkilled to me. A string would have been enough IMO.

Reply:
So we use ChunkID message only in DeleteChunk rpc as
` rpc DeleteChunks(ChunkID) returns (Nothing); `
I will replace this into  
` rpc DeleteChunks(string) returns (Nothing); `

Reply:
👍